framework/behaviors/SluggableBehavior.php

```diff
@@ -171,6 +171,10 @@ protected function isNewSlugNeeded()
             return false;
         }
 
+        if ($this->attribute === null) {
```

Comment:
When the slug is string-based and it's value is equal to the `$value` we can return `false` and slug value won't be recalculated.